src/llama-vocab.h

```diff
@@ -106,13 +113,14 @@ llama_token llama_token_eot_impl   (const struct llama_vocab & vocab);
 llama_token llama_token_eom_impl   (const struct llama_vocab & vocab);
 
 int32_t llama_tokenize_impl(
-        const struct llama_vocab & vocab,
-                      const char * text,
-                         int32_t   text_len,
-                     llama_token * tokens,
-                         int32_t   n_tokens_max,
-                            bool   add_special,
-                            bool   parse_special);
+            const struct llama_vocab & vocab,
+                       llm_tokenizer * tokenizer,
```

Comment:
`vocab` is already inside `tokenizer`, so I think we can remove it here.

Also, does `const llm_tokenizer * tokenizer` works?

Reply:
Resolved,  add `const` to  `llm_tokenizer * tokenizer`, I have updated.